kivy/core/window/__init__.py

```diff
@@ -62,11 +62,12 @@ class Keyboard(EventDispatcher):
         # specials keys
         'backspace': 8, 'tab': 9, 'enter': 13, 'rshift': 303, 'shift': 304,
         'alt': 308, 'rctrl': 306, 'lctrl': 305,
-        'super': 309, 'alt-gr': 307, 'compose': 311, 'pipe': 310,
+        'lmeta': 309, 'rmeta': 310, 'alt-gr': 307, 'compose': 311,
```

Comment:
If by `rmeta` you mean the right `alt` key i.e. "Alt Gr", then it's already there as `alt-gr` and therefore it's already there and you just broke `pipe` key name. About the `super` → `lmeta` change, this will break the key for everyone who used `if key == 'super'` as people most likely won't use the `int` value directly.

If you didn't mean by `rmeta` the "Alt Gr" key, please show how did you get it (pic of keyboard key or something), because when I press "Alt Gr", it returns `alt-gr` and its values, not `rmeta`.